easybuild/easyblocks/n/numpy.py

```diff
@@ -166,6 +166,21 @@ def get_libs_for_mkl(varname):
         if fft:
             extrasiteconfig += "\n[fftw]\nlibraries = %s" % fft
 
+        suitesparseroot = get_software_root('SuiteSparse')
+        if suitesparseroot:
+            amddir = os.path.join(suitesparseroot, 'AMD')
+            umfpackdir = os.path.join(suitesparseroot, 'UMFPACK')
```

Comment:
@boegel If the remark is about "add the os.path.exists check", where did you mention it?